.circleci/config.yml

```diff
@@ -19,12 +19,12 @@ jobs:
       - restore_cache:
           name: Restore Maven repo cache
           keys:
-            - mvn-cache-{{ checksum "pom.xml" }}
+            - mvn-cache-{{ checksum "checkstyle-tester/pom.xml" }}
```

Comment:
Caches only live for ~2 weeks in CircleCI anyway; using the sha of this file just helps in case we make major changes there.

Reply:
I recommend finding another POM since this will be gone with the CLI switch.

Reply:
done